result/nd.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.1.0
+
+info:
+  title: Neighbor Discovery table models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Nd.Entries.Request:
+      description: >-
+        The request to retrieve Neighbor Discovery entries for IPv6 interface(s).
+      type: object
+      properties:
+        ipv6_names:
+          description: >-
+            The names of IPv6 interfaces for which ND entries will be retrieved.
+            If no interface names are specified then the result set will contain all ND entries for all available IPv6 interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Ipv6/properties/name"
+        column_names:
+          description: >-
+            Name of the columns (as list), that will present in the returned result set.
+            If the column names is empty then all columns will be returned.
+            The column ipv6_name (the name of the IPv6 interface) will always be present in the returned result set.
+          type: array
+          items:
+            type: string
+            enum:
+              - ipv6_name
+              - neighbor_ip
+              - link_layer_address
+
+    Nd.Entries.Response:
+      description: >-
+        A container for Neighbor Discovery entries.
+      type: object
+      properties:
```

Comment:
Why don't we just add `resolved` as a column name where its value (true / false) for any given row should indicate the same info ?

IMHO, additional nesting of resolved vs unresolved is redundant here.